playbook/app/pb_kits/playbook/pb_fixed_confirmation_toast/_fixed_confirmation_toast.scss

```diff
@@ -47,6 +47,15 @@ $confirmation_toast_colors: (
       .pb_icon {
         color: $white;
       }
+
+      &[class*=_multi_line]  {
+        .pb_fixed_confirmation_toast_text {
```

Comment:
You can move this class in line with the parent: 

`&[class*=_multi_line]  .pb_fixed_confirmation_toast_text {`

Reply:
Done and done! Thanks for all the help!